experimental/generated/csv/fields.csv

```diff
@@ -1216,7 +657,6 @@ ECS_Version,Indexed,Field_Set,Field,Type,Level,Normalization,Example,Description
 8.3.0-dev+exp,true,process,process.hash.sha512,keyword,extended,,,SHA512 hash.
 8.3.0-dev+exp,true,process,process.hash.ssdeep,keyword,extended,,,SSDEEP hash.
 8.3.0-dev+exp,true,process,process.hash.tlsh,keyword,extended,,,TLSH hash.
-8.3.0-dev+exp,true,process,process.interactive,boolean,extended,,True,Whether the process is connected to an interactive shell.
```

Comment:
I think we still want `process.interactive`

https://github.com/elastic/endpoint-package/blob/49d9a9ac02762afa3ad158febe5edc164883d715/custom_subsets/elastic_endpoint/process/linux_event_model_event.yaml#L34

Reply:
Good catch - addressed in https://github.com/elastic/ecs/pull/1847/commits/fec3da59c2089dc1b093eae576fb39018278f977